.gitignore

```diff
@@ -35,3 +35,4 @@ docs/index.md
 
 # jenv
 /.java-version
+kotlin-js-store
```

Comment:
![image](https://user-images.githubusercontent.com/10363352/153012863-029fceb1-f5b5-417a-9803-f4ff7feafffd.png)

Should we ignore this file?

Reply:
No. It is supposed to be checked in for reproducible builds.

Reply:
Reverted.